buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -225,35 +656,44 @@
       "value": null,
       "isSecret": true
     },
-    "PB_VsoRepoUrl": {
-      "value": "--branch $(PB_Branch) https://$(PB_VsoAccountName):$(PB_VsoPassword)@devdiv.visualstudio.com/DevDiv/_git/DotNet-Core-Setup-Trusted"
-    },
     "PB_VsoAccountName": {
       "value": "dn-bot"
     },
     "PB_VsoPassword": {
       "value": null,
       "isSecret": true
     },
+    "PB_Branch": {
```

Comment:
Ah, I see where you're coming from.  `SourceBranch` is used to control VSTS' branch cloning, and `PB_SourceBranch` is currently used for our own clone command.  For the most part, we ignore `SourceBranch` since otherwise you have to pay extra careful attention to ensure that PB_SourceBranch and Branch align and the clone url + git's repository align.